rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -304,10 +297,243 @@ namespace rocal{
         m.def("rocalRelease",&rocalRelease);
         // rocal_api_types.h
         py::class_<TimingInfo>(m, "TimingInfo")
-            .def_readwrite("load_time",&TimingInfo::load_time)
-            .def_readwrite("decode_time",&TimingInfo::decode_time)
-            .def_readwrite("process_time",&TimingInfo::process_time)
-            .def_readwrite("transfer_time",&TimingInfo::transfer_time);
+            .def_readwrite("load_time", &TimingInfo::load_time)
+            .def_readwrite("decode_time", &TimingInfo::decode_time)
+            .def_readwrite("process_time", &TimingInfo::process_time)
+            .def_readwrite("transfer_time", &TimingInfo::transfer_time);
+        py::class_<rocalTensor>(m, "rocalTensor")
+                .def(
+                "batch_height",
```

Comment:
return output_tensor.info().max_shape().at(1); 
We return max_shape() at 1for height & 0 for width - which would change for NCHW & NHWC

Reply:
It would return samples & channels here.
Yes , max_shape() referring to Width and height should be same for both NHWC and NCHW

Reply:
Please rename this to max shape itself since width and height will not make sense.
or else return max_shape as a list and we can get width and height / samples and channels.

Reply:
@swetha097 Please address this

Reply:
@swetha097 Please address this.If needed get Fiona's help.

Reply:
Made to return the max_shape